scania-digital-design-system/tegel
# feat(toast): prefix change

**Describe pull-request**  
Prefix change in Toast component

**Solving issue**  
Fixes: DTS-1621

**How to test**  
1. Go to the Storybook link below
2. Check if the component looks and feels as in the main
3. Check if the prefix is changed in the code too

**Suggested test steps**
